src/debug.c

```diff
@@ -381,39 +381,88 @@ void mallctl_string(client *c, robj **argv, int argc) {
 void debugCommand(client *c) {
     if (c->argc == 2 && !strcasecmp(c->argv[1]->ptr,"help")) {
         const char *help[] = {
-"ASSERT -- Crash by assertion failed.",
-"CHANGE-REPL-ID -- Change the replication IDs of the instance. Dangerous, should be used only for testing the replication subsystem.",
-"CRASH-AND-RECOVER <milliseconds> -- Hard crash and restart after <milliseconds> delay.",
-"DIGEST -- Output a hex signature representing the current DB content.",
-"DIGEST-VALUE <key-1> ... <key-N>-- Output a hex signature of the values of all the specified keys.",
-"DEBUG PROTOCOL [string|integer|double|bignum|null|array|set|map|attrib|push|verbatim|true|false]",
-"ERROR <string> -- Return a Redis protocol error with <string> as message. Useful for clients unit tests to simulate Redis errors.",
-"LOG <message> -- write message to the server log.",
-"LEAK <string> -- Create a memory leak of the input string.",
```

Comment:
@oranagra I found this was deleted in help message, is that on purpose?

I also found out: `debug protocol bignum` will crash the redis-cli
```
127.0.0.1:6379> debug protocol bignum
Error: Protocol error, got "(" as reply type byte
```

I suppose the `(` should be a `+`
```
        } else if (!strcasecmp(name,"bignum")) {
            addReplyProto(c,"(1234567999999999999999999999999999999\r\n",40);
```

Reply:
@enjoy-binbin if any DEBUG help entries were removed by this PR it wasn't intentional (it was suppose to just be a formatting PR).
feel free to make a PR to re-introduce them.

regarding bignum, it's just a part of the RESP3 protocol that's not yet used by redis, and not yet supported by redis-cli.
see https://github.com/antirez/RESP3/blob/master/spec.md

Reply:
@oranagra 
oh i got it now. Thanks for the tip. 
I integrated it with some other cleanups into a pr https://github.com/redis/redis/pull/9098